build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.code-quality.gradle.kts

```diff
@@ -42,18 +45,28 @@ val rules by configurations.creating {
     }
 }
 
+// FIXME the below changes are ineffective - the Worker API will classload the lib directory of the current Gradle distribution
```

Comment:
Does this comment apply only to the commented out isBundleGroovy4 val, or to the whole detection process below?

Reply:
I'm not 100% sure anymore - that's because I found some strange behavior in `CodeNarcPlugin#DEFAULT_CODENARC_VERSION` when I tried to derive the value, rather than hardcode a String.  Let's discuss on Slack.

Reply:
I've removed these comments - I think the trouble was caused by our custom classloader and ASM issues we discussed earlier today.